include/hisi_qm_udrv.h

```diff
@@ -51,6 +53,7 @@ struct hisi_qm_queue_info {
 	__u16 hw_type;
 	bool cqc_phase;
 	pthread_spinlock_t lock;
+	unsigned long qfrs_offset[UACCE_QFRT_MAX];
```

Comment:
BTW, why we need qfrs_offset, we can set ds_tx_base and ds_rx_base directly.

And what is ds_tx_base/ds_rx_base, please get a meanful name, and add comments here